js/accounts/pocketbase.js

```
@@ -156,7 +156,8 @@ const syncManager = {
                             (jsFriendly.trim().startsWith('[') || jsFriendly.trim().startsWith('{')) &&
                             !jsFriendly.match(/function|=>|window|document|alert|eval/)
                         ) {
-                            return new Function('return ' + jsFriendly)();
+                            //TODO: maybe this could be parsed as JSON?
+                            return () => eval(jsFriendly);
```

Biome (2.4.9)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)